app/lib/handleZcapRequest.ts

```diff
@@ -0,0 +1,96 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { WAS_KEYS } from '../../app.config';
+import { Ed25519VerificationKey2020 } from '@digitalcredentials/ed25519-verification-key-2020';
+// @ts-ignore
+import { ZcapClient } from '@digitalbazaar/ezcap';
+import { Ed25519Signature2020 } from '@digitalcredentials/ed25519-signature-2020';
+
+import { displayGlobalModal } from './globalModal';
+
+
+interface ZcapReq {
+    query: {
+      type: string;
+      capabilityQuery: {
+        allowedAction: string[];
+        controller: string;
+        invocationTarget: {
```

Comment:
`invocationTarget` can also be a String (a simple URL), or the object outlined here.